resources/tentative/charts/dist/observable-plot.html

```diff
@@ -4,7 +4,8 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Observable Plot</title>
-      <script type="module" crossorigin src="./assets/plot-e9515604.js"></script>
+      <script type="module" crossorigin src="./assets/plot-92055dcb.js"></script>
+      <link rel="modulepreload" crossorigin href="./assets/flights-airports-9a9e6422.js">
```

Comment:
Unfortunately I can't control the indentation generated by vite. One solution could be to run prettier after the build, but I'm not sure it's worth it. What do you think?

Reply:
This also comes from the fact that this html file has been configured according to the prettier rule (with 4 spaces) but the chartjs one hasn't (so it's 2 spaces). I believe vite outputs 2 spaces because that's the most common configuration by developers.

I don't think this is so important because it's a generated file. But tell me what you think.

Reply:
oh I see. I'm fine with this if it's autogenerated.